src/components/CivicProfileForms/HousingInfo.jsx

```diff
@@ -47,48 +50,106 @@ const HousingInfo = () => {
   };
 
   return (
-    <form onSubmit={handleSubmit}>
+    <form
+      onSubmit={handleSubmit}
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '8px',
+        justifyContent: 'space-between'
+        // maxWidth: '720px',
+      }}
+    >
       <div
         style={{
-          padding: '8px',
           display: 'flex',
-          flexDirection: 'column',
           gap: '8px',
-          maxWidth: '360px'
+          minWidth: '846px',
+          padding: '8px'
+        }}
+      >
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '8px',
+            padding: '8px',
+            width: '40%'
+          }}
+        >
+          <TextField
+            id="street-input"
+            name="lastPermanentStreet"
+            label="Street:"
+            onChange={handleChange}
+            value={formData.lastPermanentStreet}
+          />
+          <TextField
+            id="city-input"
+            name="lastPermanentCity"
+            label="City:"
+            onChange={handleChange}
+            value={formData.lastPermanentCity}
+          />
+          <TextField
+            id="state-input"
+            name="lastPermanentState"
+            label="State:"
+            onChange={handleChange}
+            value={formData.lastPermanentState}
+          />
+          <TextField
+            id="zip-input"
+            name="lastPermanentZIP"
+            label="ZIP Code:"
+            onChange={handleChange}
+            value={formData.lastPermanentZIP}
```

Comment:
Would want to add some kind of client-side validation for zip code to accept only numeric values.

An exception could be made for `-` for extended zip codes with patterns like `/^\d{5}-\d{4}$/`, where it starts with 5 initial digits and ends with 4 digits after the dash. But we could add this pattern for validation in the future if we want to.

Reply:
Per @leekahung in PR #633 note, added form validation to `lastPermanentZIP` `<TextField>`, added `zipError, setZipError` in `useState`, a conditional check in `handleChange()` & `error` & `helperText` fields to the `<TextField>` in this branch.

Reply:
Had just checked this one more time to see if everything's in good order. It looks like validation isn't working entirely.

https://github.com/codeforpdx/PASS/assets/14917816/30a51f2d-ecc9-4e3f-8d40-0106443914ef

As shown the input is wrong, but I could submit the form regardless of the error.

The errors are shown correctly with the TextField, but the submissions and notifications are not. We could, of course, have a separate issue opened up to implement the correct notifications, but I think we could add the condition for `zipError` in this PR to prevent form submission, (we're already doing it for the states `isSuccess` and `storedDataset`).

Of course, there's also the case where we can consider an empty value as a valid input since the TextFields here are all optional so I'm thinking we could include an empty string as the other condition where the pattern is okay.